TSC/core-and-hosted-projects.md

```diff
@@ -20,34 +20,41 @@ Core projects must further the mission of the Bytecode Alliance and be seen as v
+  * An update to `/projects/README.md` to include an entry for the project.
 
 The Bytecode Alliance Technical Steering Committee (TSC) will then perform the following steps:
 
 * [ ] Provide timely acknowledgment of the application.
 * [ ] Perform an in-depth review of the project, evaluate whether it meets the hosted project requirements, and decide based on consensus whether to accept the project.
-* [ ] Reply in the issue with the results of the review, providing any feedback they might have, and whether the project has been accepted.
-* [ ] If it has been accepted, then the TSC will work with maintainers on the logistics of moving the repository to the `bytecodealliance` github organization.
+* [ ] Reply in the Pull Request with the results of the review, providing any feedback they might have, and whether the project has been accepted.
+* [ ] If it has been accepted, then the TSC will work with maintainers on the logistics of moving the repository to the `bytecodealliance` github organization and merge the Pull Request once that process is complete.
 
 ## Applying for Promotion from Hosted Project to Core Project
 
 Once the project maintainers have come to a consensus agreement that they wish for their project's promotion to a core project, they may initiate the application process by performing the following steps:
 
-* [ ] Create an issue within the [the Bytecode Alliance governance repository](https://github.com/bytecodealliance/governance) using [the "New Core Project" issue template](https://github.com/bytecodealliance/governance/issues/new/choose).
-  * Fill in all of the `TODO`s.
-  * Do not make unsubstantiated claims; provide evidence. For example, if you claim that your project has a CI job running tests under [AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html), provide links to its CI configuration, test infrastructure, and bugs caught.
+* [ ] Create a Pull Request against [the Bytecode Alliance governance repository](https://github.com/bytecodealliance/governance) with the following changes:
+  * a copy of [/templates/projects/proposal-core-project.md](../templates/projects/proposal-core-project.md), with all of the `TODO`s filled in, as a new file under `/projects/core/$PROJECT_NAME/proposal-core.md`
+    * Do not make unsubstantiated claims; provide evidence. For example, if you claim that your project has a CI job running tests under [AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html), provide links to its CI configuration, test infrastructure, and bugs caught.
+  * moving any existing files from `/projects/hosted/$PROJECT_NAME` to `/projects/core/$PROJECT_NAME`
+    * If a `proposal.md` exists for the project, move it to `/projects/core/$PROJECT_NAME/proposal-hosted.md`
+  * An update to `/projects/README.md` to move the project to the list of core projects.
```

Comment:
nit: I personally would have chosen to use paths relative to the project root, e.g., `projects/README.md`... but not a significant issue.

Reply:
I guess now that #116 has landed I can just turn this into a link?